### ISSUE TITLE
Case insensitive comparison option

### ISSUE THREAD

**Issue by tip2tail** [ryu1kn/vscode-partial-diff#10]
Hello

The language I work with is case insensitive.  As such when I compare two programs I get a lot of false positives in the diff due to text case.

As a feature request, the ability to either exclude or, preferably, highlight a different colour, any case-only diffs.

Thanks
t2t

**Comment by ryu1kn** [comment]
I see, languages like SQL...

This extension is internally using `vscode.diff` command and the command doesn't support case insensitive text comparison nor allow you to specify different colours. If you really need it, you might want to open an issue on vscode repo.

Meanwhile, we could convert all characters to lower case just before we compare 2 programs but then the characters you'll see in the diff view are all lower case...

**Comment by tip2tail** [comment]
I have raised this now - https://github.com/Microsoft/vscode/issues/32905

**Comment by ryu1kn** [comment]
Thanks @tip2tail , I'm closing this issue for now then.

**Comment by ryu1kn** [comment]
Hi @tip2tail , I've just published v0.5.0 which allows you to normalise texts just before you feed them into the diff view. I'm wondering if you can use this feature to ignore case-only diffs by normalising all characters in the both texts to lower case (or upper case).

```json
"partialDiff.preComparisonTextNormalizationRules": [
  {
    "name": "Capitalise",
    "match": ".*",
    "replaceWith": {"letterCase": "lower"}
  }
]
```
